policyengine_us/parameters/gov/states/ct/tax/income/rebate/amount.yaml

```diff
@@ -0,0 +1,12 @@
+description: Connecticut provides tax rabate at this amount per child
+metadata:
+  period: year
+  unit: currency-USD
+  label: Connecticut child tax rebate amount
+  reference:
+    - title: 2022 Child Tax Rebate
+      href: https://portal.ct.gov/drs/credit-programs/child-tax-rebate/overview
```

Comment:
Could you look through the pdf and see if it mentions it:
https://cga.ct.gov/2022/ACT/PA/PDF/2022PA-00118-R00HB-05506-PA.PDF